cg/meta/workflow/nf_analysis.py

```diff
@@ -174,6 +174,15 @@ def write_params_file(self, case_id: str, workflow_parameters: dict) -> None:
             file_path=self.get_params_file_path(case_id=case_id),
         )
 
+    def write_nextflow_config(self, case_id: str) -> None:
+        """Write nextflow config in json format."""
+        if self.get_nextflow_config_content:
+            LOG.debug("Writing nextflow config file")
+            write_txt(
```

Comment:
fair enough

Reply:
the txt parser is modified in a general way so it can write from lists or strings. It's has an expanded functionality but it's still very general, nothing config-specific, so this should remain and doesn't have to be cleaned up. 
What this PR adds that will have to be updated is the `get_nextflow_config_content` method, which instead of returning a simple string, will have to parse and concat the servers configs

Reply:
resolving this conversation